apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -57,7 +57,7 @@ public static function get_instance() {
 	/**
 	 * Register post types.
 	 */
-	public function register_template_post_types() {
+	public static function register_template_post_types() {
```

Comment:
This should not be `static` as it's already being referenced in `Full_Site_Editing::_construct` under `$this`

You'd probably do just fine calling `Full_Site_Editing::get_instance()` in `a8c_fse_on_plugin_activation`, or you could alternately move all the functions in `__construct()` into a separate `init` method that you call appropriately, and then just do `Full_Site_Editing::get_instance()->init()` when you need to initialize and `Full_Site_Editing::get_instance()->register_template_post_types()` when you just want that method

Reply:
Good catch! Updated in 709a93b34e4dcfccf502f4184875a650712e72b6.